Common/Utils/BossRushUtils.cs

```diff
@@ -174,7 +165,13 @@ public static partial class BossRushUtils {
 					npc.Add(Npc);
 			}
 		}
-		public static int StatModifierDamageValue(this StatModifier modifier) => (int)(modifier.Base * modifier.Additive * modifier.Multiplicative + modifier.Flat);
+		public static int ToMinute(float minute) => (int)(ToSecond(60) * minute);
+		public static int ToSecond(float second) => (int)(second * 60);
+		public static int ToIntValue(this StatModifier modifier) => (int)((modifier.Base + 1) * modifier.Additive * modifier.Multiplicative + modifier.Flat);
+		public static float ToFloatValue(this StatModifier modifier, float additionalMulti = 1, int round = -1) 
+			=> round == -1 ?  
+			((modifier.Base + 1) * modifier.Additive * modifier.Multiplicative + modifier.Flat) * additionalMulti :
```

Comment:
Any reason why not just `modifier.ApplyTo(1) * additionalMulti`?

Reply:
wait it is the same ?